methods/transient/Transient_approach2.2.ipynb

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
<p>This is a good start on an explanation of what analysis is done in the notebook. I would put this in the first cell instead of the second. It also needs to mention the fact that user has to run the&nbsp;<code>Transient_base</code>&nbsp;notebook first.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>